libs/portals/admin/regulations-admin/src/screens/Edit.tsx

```diff
@@ -0,0 +1,186 @@
+  useLawChaptersQuery,
+  useMinistriesQuery,
+  useRegulationDraftQuery,
+} from '../utils/dataHooks'
+import { PortalModuleComponent } from '@island.is/portals/core'
+import { MessageDescriptor } from '@formatjs/intl'
+import { editorMsgs } from '../lib/messages'
+import { EditBasics } from '../components/EditBasics'
+import { EditMeta } from '../components/EditMeta'
+import { EditSignature } from '../components/EditSignature'
+import { EditImpacts } from '../components/EditImpacts'
+import { EditReview } from '../components/EditReview'
+import { EditPublish } from '../components/EditPublish'
+import {
+  Box,
+  GridColumn,
+  GridRow,
+  Text,
+  toast,
+} from '@island.is/island-ui/core'
+import { SaveDeleteButtons } from '../components/SaveDeleteButtons'
+import { DraftingNotes } from '../components/DraftingNotes'
+import { ButtonBar } from '../components/ButtonBar'
+
+// ---------------------------------------------------------------------------
+
+const assertStep = (maybeStep: string | undefined): Step => {
+  const stepName = ensureStepName(maybeStep || 'basics')
+  if (stepName) {
+    return stepName
+  }
+  throw new Error('Invalid RegulationDraft editing Step')
+}
+
+const assertDraftId = (maybeId: string | undefined): RegulationDraftId => {
+  if (typeof maybeId === 'string' && isUuid(maybeId)) {
+    return maybeId as RegulationDraftId
+  }
+  throw new Error('Invalid RegulationDraft editing Id')
+}
+
+const assertImpactId = (maybeId: string | undefined): DraftImpactId => {
+  if (typeof maybeId === 'string' && maybeId && isUuid(maybeId)) {
+    return maybeId as DraftImpactId
+  }
+  throw new Error('Invalid DraftImpactId')
+}
+
+// ---------------------------------------------------------------------------
+
+const stepData: Record<
+  Step,
+  {
+    title: MessageDescriptor
+    intro?: MessageDescriptor
+    Component: () => ReturnType<FC>
+  }
+> = {
+  basics: {
+    title: editorMsgs.stepContentHeadline,
+    Component: EditBasics,
+  },
+  signature: {
+    title: editorMsgs.stepSignatureHeadline,
+    intro: editorMsgs.stepSignatureIntro,
+    Component: EditSignature,
+  },
+  meta: {
+    title: editorMsgs.stepMetaHeadline,
+    Component: EditMeta,
+  },
+  impacts: {
+    title: editorMsgs.stepImpactHeadline,
+    intro: editorMsgs.stepImpactIntro,
+    Component: EditImpacts,
+  },
+  review: {
+    title: editorMsgs.stepReviewHeadline,
+    intro: editorMsgs.stepReviewIntro,
+    Component: EditReview,
+  },
+  publish: {
+    title: editorMsgs.stepPublishHeadline,
+    intro: editorMsgs.stepPublishIntro,
+    Component: EditPublish,
+  },
+}
+
+const EditScreen = () => {
+  const t = useLocale().formatMessage
+  const { draft, error: errorSate, step: stepState } = useDraftingState()
+  const step = stepData[stepState.name]
+
+  useEffect(() => {
+    if (errorSate) {
+      const { message, error } = errorSate
+      console.error(error || message)
+      toast.error(t(message))
+    }
+  }, [errorSate, t])
```

Comment:
Typo. errorSate -> errorState
```suggestion
const EditScreen = () => {
  const t = useLocale().formatMessage
  const { draft, error: errorState, step: stepState } = useDraftingState()
  const step = stepData[stepState.name]
  useEffect(() => {
    if (errorState) {
      const { message, error } = errorState
      console.error(error || message)
      toast.error(t(message))
    }
  }, [errorState, t])
```